jdi-light-vuetify-tests/src/main/java/io/github/com/pages/BreadcrumbsPage.java

```diff
@@ -0,0 +1,46 @@
+package io.github.com.pages;
+
+import com.epam.jdi.light.vuetify.annotations.JDIBreadcrumbs;
+import com.epam.jdi.light.vuetify.elements.complex.Breadcrumbs;
+
```

Comment:
Need link to vuetify page https://vuetifyjs.com/en/components/breadcrumbs/

Reply:
It is in `jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/Breadcrumbs.java`